resolwe/flow/utils/__init__.py

```diff
@@ -24,7 +25,7 @@
 def get_data_checksum(proc_input, proc_slug, proc_version):
     """Compute checksum of processor inputs, name and version."""
     checksum = hashlib.sha1()
```

Comment:
This is even more relevant since [Google uncovered the first practical attack on SHA1](https://security.googleblog.com/2017/02/announcing-first-sha1-collision.html).